web/client/themes/default/less/swipe.less

```diff
@@ -13,11 +13,42 @@
 }
 
 .mapstore-swipe-settings {
+
+    .mapstore-swipe-settings-modal {
+        min-height: 22% !important;
+        height: auto !important;
+        width: 25% !important;
+        overflow: visible !important;
```

Comment:
you should avoid `!important` as much as possible. They should be used only to override some inline styles. Otherwise please use css rules priority system to make these rules win over the others